index.bs

```diff
@@ -589,16 +589,16 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		  Promise&lt;void> writeText(DOMString data);
 		};
 
-		typedef (DOMString or Blob) ClipboardItemDataType;
-		typedef Promise&lt;ClipboardItemDataType> ClipboardItemData;
+		typedef (DOMString or Blob) ClipboardItemValue;
+		typedef Promise&lt;ClipboardItemValue> ClipboardEntryValuePromise;
 
-		callback ClipboardItemDelayedCallback = ClipboardItemData ();
+		callback ClipboardItemDelayedCallback = ClipboardEntryValuePromise ();
 
 		[Exposed=Window] interface ClipboardItem {
-		  constructor(record&lt;DOMString, ClipboardItemData> items,
+		  constructor(record&lt;DOMString, ClipboardEntryValue> entries,
```

Comment:
This change assumes that https://github.com/w3c/clipboard-apis/pull/126 is correct (e.g. matching Chrome's current behaviour). If this should accept `Promise`s, then either:

- the record values should be `ClipboardEntryValuePromise` and Chrome's implementation should be changed, or
- the record values should be allowed to be either `ClipboardEntryValue` or `ClipboardEntryValuePromise`.